app/views/admin/users/_list.html.erb

```diff
@@ -10,7 +10,15 @@
   <tbody>
     <% @users.each do |user| %>
       <tr>
-        <td><%= user.organization&.name %></td>
+        <td>
+          <% user.roles.each do |role| %>
+            <% if role.resource_type == 'Organization' %>
+              <%= role.resource.name %>
+            <% elsif role.resource_type == 'Partner' %>
+              , <%= role.resource.name %>
```

Comment:
I think we're banking on the order of the resources. Probably best to just join whatever we see, e.g.

```ruby
<%= user.roles.map { |r| r.resource.try(:name).presence || r.resource_id}.join(', ') %>
```